internal/tfengine/tfengine.go

```diff
@@ -157,16 +157,12 @@ func dump(conf *Config, pwd, cacheDir, outputPath string, wantedTemplates map[st
 func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateInfo) error {
 	outputPath = filepath.Join(outputPath, ti.OutputPath)
 
-	data := make(map[string]interface{})
+	data := template.CopyData(conf.Data)
 
 	// Make the schema available.
 	trimDescriptions(conf.Schema)
 	data["__schema__"] = conf.Schema
 
-	if err := template.MergeData(data, conf.Data); err != nil {
```

Comment:
Could you explain why MergeData does not work in this case for the `labels` map for the project recipe, but `labels` map for other resources, e.g. GKE works and other fields in the project recipe also works?

Reply:
Answered in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/1079#discussion_r774024687